
  0%|          | 0/38 [00:00<?, ?it/s]/home/curie/miniconda3/envs/sb/lib/python3.10/site-packages/torch/nn/modules/conv.py:306: UserWarning: Attempt to open cnn_infer failed: handle=0 error: libcudnn_cnn_infer.so.8: cannot open shared object file: No such file or directory (Triggered internally at ../aten/src/ATen/native/cudnn/Conv_v8.cpp:78.)
  return F.conv1d(input, weight, bias, self.stride,
































100%|██████████| 38/38 [01:12<00:00,  1.90s/it]
Epoch Number::   0%|          | 0/10 [00:00<?, ?it/s]
Training Batch:   0%|          | 0/150 [00:00<?, ?it/s]






















































